website/content/docs/secrets/pki/considerations.mdx

```diff
@@ -318,6 +309,72 @@ revoked and expired certificates can be configured automatically via the
 `/config/auto-tidy` endpoint. Both of these should be enabled to ensure
 compatibility with the wider PKIX ecosystem and performance of the cluster.
 
+## Spectrum of Revocation Support
+
+Starting with Vault 1.13, the PKI engine has the ability to support a
+spectrum of cluster sizes and certificate revocation quantities.
+
+For users with few revocations or who want a unified view and have the
+inter-cluster bandwidth to support it, we recommend turning on auto
+rebuilding of CRLs, cross-cluster revocation queues, and cross-cluster CRLs.
+This allows all consumers of the CRLs to have the most accurate picture of
+revocations, regardless of which cluster they talk to.
+
+If the unified CRL becomes to big for the underlying storage mechanism or
+for a single host to build, we recommend relying on OCSP instead of CRLs.
+These have much smaller storage entries, and the CRL `disabled` flag is
+independent of `unified_crls`, allowing unified OCSP to remain.
+
+However, when cross-cluster traffic becomes too high (or if CRLs are still
+necessary in addition to OCSP), we recommend sharding the CRL between
+different clusters. This has been the default behavior of Vault, but with
+the introduction of per-cluster, templated AIA information, the leaf
+certificate's Authority Information Access (AIA) info will point directly
+to the cluster which issued it, allowing the correct CRL for this cert to
+be identified by the application. This more correctly mimics the behavior
+of [Let's Encrypt's CRL sharding](https://letsencrypt.org/2022/09/07/new-life-for-crls.html).
+
+This sharding behavior can also be used for OCSP, if the cross-cluster
```

Comment:
```suggestion
This sharding behavior can also be used for OCSP when the cross-cluster
```

Reply:
I think this is more of an `if`, not a `when`. Not all customers will have enough PKI demand on a given mount that they'll necessarily hit cross-cluster bandwidth limitations.